process/sync/shardblock.go

```diff
@@ -649,16 +653,28 @@ func (boot *ShardBootstrap) SyncBlock() error {
 	}
 
 	if boot.isForkDetected {
-		log.Info(fmt.Sprintf("fork detected at nonce %d with hash %s\n",
-			boot.forkNonce,
-			core.ToB64(boot.forkHash)))
+		isForcedFork := boot.forkNonce == math.MaxUint64 && boot.forkHash == nil
```

Comment:
duplicate code found here and in metablock.go L426:L447

Reply:
refactored